p/scripts/main.js

```diff
@@ -1286,7 +1313,57 @@ function init_crypto_form() {
 }
 //</crypto form (Web login)>
 
+var sidebar = document.getElementById('sidebar');
+var useJsScrollbar = true;
+try {
+	useJsScrollbar = sidebar && !CSS.supports('scrollbar-color: auto') &&
+		!(parseInt(getComputedStyle(sidebar, '::-webkit-scrollbar').width) < sidebar.offsetWidth);
```

Comment:
Excellent. Now that'll be fully comprehensible in five years without extensive use of git blame. :-)